translations/monero-GUI-guide_es.md

```diff
@@ -0,0 +1,331 @@
+**(6) Dirección:** Esta es la dirección de recepción del pago que estás intentando verificar.    
+**(7) Mensaje (opcional):** Este es el mensaje opcional que puede haber sido incluido junto con la prueba.
+**(8) Firma:** Esta es la firma generada para probar un pago.    
+**(9) Verificar:** Haz clic aquí una vez que hayas introducido la información anterior para verificar si la prueba de la transacción facilitada es válida.
+
+## 5.3. RingDB compartido
+
+![shared ringdb](/media/black_sharedringdb.png)
+
+## 5.4. Firmar - verificar
+
+La pestaña de `Firmar/verificar` facilita herramientas para firmar un mensaje o fichero con tu clave privada y verificar la autenticidad del mensaje o fichero.
+
+![sign/verify](/media/black_sign-verify.png)
+
+### 5.4.1. Firma
+
+**(1) Mensaje:** Aquí es donde se introduce un mensaje para firmar.    
+**(2) Mensaje del archivo:** Aquí se puede elegir un archivo para firmar. Haz clic en `Examinar` para navegar por el sistema de archivos.    
+**(3) Firma:** Aquí es donde se mostrará la firma única una vez que se haya hecho clic en el botón de `Firmar` de los apartados anteriores. La firma del mensaje o archivo que se haya facilitado es realizada con la clave privada del monedero. Esta firma se pueded entregar como prueba junto con el mensaje o archivo en cuestión.
+
+### 5.4.2. Verifica
+
+**(4) Verificar mensaje:** Aquí es donde se coloca un mensaje que ha sido firmado.    
+**(5) Verificar archivo:** Aquí es donde se ingresa la ruta a un archivo que se ha firmado. Haz clic en `Examinar` para navegar por el sistema de archivos.    
+**(6) Dirección:** Aquí es donde se introduce la dirección pública de Monero del firmante.    
+**(7) Firma:** Aquí es donde se introduce la firma que está verificando. Una vez que se haya ingresado toda la información requerida, es necesario hacer clic en el botón de `Verificar`. Una ventana emergente certificará si la firma es válida.
+
+# 6. Ajustes
+
+La pestaña de `Ajustes` facilita herramientas para configurar las diferentes opciones de la aplicación.
+
+![settings](/media/black_settings.png)
+
+**(1) Cerrar billetera:** Cierra el monedero y reinicia el asistente de creación.    
+**(2) Crear cartera solo de vista:** Crea una versión de solo visualización de la cartera actual. Se te solicitará un nombre de archivo y una contraseña para cifrarla.    
+**(3) Volver a escanear el saldo de la billetera:** Haz un nuevo escaneo de las salidas gastadas.    
+**(4) Cambiar contraseña:** Cambia la contraseña de su billetera.    
+**(5) Nodo local:** Usa un nodo local y descarga la _blockchain_. Puedes usar un nodo de arranque hasta que tu cadena de bloques esté sincronizada.    
+**(6) Nodo remoto:** Usa un nodo remoto sin descargar la cadena de bloques.    
+**(7) Dirección de inicio / nodo remoto:** Introduce el nombre del _host_ o la dirección IP del nodo remoto.    
+**(8) Bootstrap / Remote Node Port:** Introduce el puerto del nodo remoto.    
```

Comment:
s/Remote Node Port, to: **"Puerto de nodo remoto"**

I believe this was missing.